app/components/live-participants.hbs

```diff
@@ -37,7 +41,7 @@
                     {{on 'click' (fn @openKickoutModal peer)}}
                   >
                     <IconifyIcon
-                      data-test-icon='remove'
+                      data-test-kickout-icon='remove'
```

Comment:
I would say change it back to data-test-icon, since remove is already passed.
It wasn't in the change before and I saw in test that you were doing data-test-icon in tests which is the wrong way to get the element also.

It will be like this in tests :
`assert.dom('[data-test-icon="remove"]')`

Reply:
I'm not sure if this actually is testing the thing or not